### PR TITLE
loungy 0.1.2 (new cask)

### PR DESCRIPTION
**Pull request by MatthiasGrandl**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---
 
[Loungy](https://github.com/MatthiasGrandl/loungy) is an open-source GPU-accelerated application launcher in the vein of Raycast and Alfred.

### CODE REVIEW

File: Casks/l/loungy.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  url "https://github.com/MatthiasGrandl/Loungy/releases/download/v#{version}/Loungy_#{version}_universal.dmg"
+  name "Loungy"
+  desc "GPU accelerated application launcher in the vein of Raycast and Alfred"
+  homepage "https://github.com/MatthiasGrandl/Loungy"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
```
Comment (bevanjkay) [inline Casks/l/loungy.rb:14]: The `github_latest` strategy is only required if you have pre-releases or tags with no corresponding release. In this case it looks like it ok to remove the `livecheck` block and to use the default `git` livecheck strategy that will be inferred automatically from the `url`.

```suggestion
```

File: Casks/l/loungy.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+cask "loungy" do
+  version "0.1.2"
+  sha256 "81700f11ed264fceae4623b906de98b52340f760baa1dede3ff5c2eeaf008320"
+
+  url "https://github.com/MatthiasGrandl/Loungy/releases/download/v#{version}/Loungy_#{version}_universal.dmg"
+  name "Loungy"
+  desc "GPU accelerated application launcher in the vein of Raycast and Alfred"
```
Comment (bevanjkay) [inline Casks/l/loungy.rb:7]:
```suggestion
  desc "Application launcher"
```

Feel free to suggest an alternative to this, but make sure it meets the [`desc guidelines`](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

**Review by bevanjkay — CHANGES_REQUESTED** [review]
I have left a couple of comments below.

My additional concern is the wording on your repository sounds to me like the product is not mature enough to be included in `homebrew-cask`. Is this statement still true?

> It isn't (yet) intended to be used outside of experimentation.

**Comment by MatthiasGrandl** [comment]
> I have left a couple of comments below.
> 
> My additional concern is the wording on your repository sounds to me like the product is not mature enough to be included in `homebrew-cask`. Is this statement still true?
> 
> > It isn't (yet) intended to be used outside of experimentation.

I updated the README.md. Loungy has matured a lot since then and quite a few people are using it already.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @MatthiasGrandl

**Pull request merged**